src/styles/corporate/styles.scss

```diff
@@ -60,79 +84,164 @@ section.cards-section > div > h3 {
   }
 }
 
+// Header override
```

Comment:
Perhaps we could tweak so that header title doesn't get chopped off on mobile (can only see subtext) ?

<img width="412" alt="image" src="https://github.com/user-attachments/assets/031aea1c-02d1-4439-8e60-a2790a4a58cd">

Reply:
Keeping only Developers Docs link on mobile fix the issue. Like said previously, api.wazo.io is a temporary website. When it will be merged inside docusaurus, mobile is well supported :)